### ISSUE TITLE
Last layer's weights when changing the number of classes + expected labels

### ISSUE THREAD

**Issue by IhamdiS** [bonlime/keras-deeplab-v3-plus#16]
Thank you for sharing your code

In my case , i want to train on my own dataset, should the labels (mask) be a 2-d image (with one chanel eventually)  where each pixel has as value the class label (for example pixel belonging to a person has value 15,ect, which will requires later on a colormap) ? or what format the labels (mask) should have.......may i know what software do you use to create your labels ?

again thank you for sharing your effort

**Comment by bhack** [comment]
For labels check original impl https://github.com/tensorflow/models/blob/master/research/deeplab/utils/input_generator.py#L133-L134

**Comment by IhamdiS** [comment]
thank you, will check it
